src/Makevars

```diff
@@ -5,19 +5,29 @@ LIBDIR = ./rust/target/$(RPOLARS_PROFILE)
 STATLIB = $(LIBDIR)/libr_polars.a
 PKG_LIBS = -L$(LIBDIR) -lr_polars
 RPOLARS_RUST_SOURCE ?= ./rust
-RPOLARS_CARGO_CLEAN_DEPS ?= nope
+
 rpolars_ext_binary = "$(RPOLARS_RUST_SOURCE)/target/$(RPOLARS_PROFILE)/libr_polars.a"
 
 all: C_clean
 
 $(SHLIB): $(STATLIB)
 
+CARGOTMP = $(CURDIR)/.cargo
+
 $(STATLIB):
+	if [ "$(NOT_CRAN)" != "true" ]; then \
+		export CARGO_HOME=$(CARGOTMP); \
```

Comment:
> If `NOT_CRAN=true`, set CARGO_HOME to a temporary directory to suppress writing to the home directory.

Shouldn't it be `"$(NOT_CRAN)" == "true"` above (and same below)? sorry if this is trivial, I'm not used to makefiles

Reply:
by the way I find it harder to read when we have double negative, like `NOT_CRAN = false`. Is it possible to rename `NOT_CRAN` to `CRAN`?

Reply:
Thanks for taking a look at this, the PR description was incorrect! (I have fixed)

The intent of this is to set `CARGO_HOME` on CRAN, so if not `NOT_CRAN=ture`, it is considered it is on CRAN.

Reply:
thanks for the explanation, re my second comment, `"$(NOT_CRAN)" != "true"` -> `"$(CRAN)" == "false"` would be much clearer

Reply:
`NOT_CRAN` is an environment variable set by packages such as `devtools` and used by the community to indicate that it is outside of CRAN.
In other words, on CRAN, no environment variable is set to indicate that they are on CRAN.

Reply:
Here is a note about the origin of `NOT_CRAN`.
https://github.com/extendr/rextendr/pull/233#issuecomment-1475259856